graphql_compiler/tests/test_helpers.py

```diff
@@ -559,6 +560,7 @@ def compare_ignoring_whitespace(
     test_case.assertEqual(transform(expected), transform(received), msg=msg)
 
 
+@lru_cache()
```

Comment:
This is a horrible hack. It just minimizes the lines of code I had to change for my data exploration. If I were to do this properly, I would change the `get_schema` function for a constant.